### DISCUSSION TITLE
EXE fails in Windows 11

### DISCUSSION THREAD

**Discussion by ardee00** [Open-Shell/Open-Shell-Menu#2074] [Q&A] [answered]
When I run the EXE installer, the error "Failed to extract setup file "C:\ProgramData\OpenShellSetup64_4_4_194.msi." appears. Tried with previous installer, same result.

**Comment by ge0rdi** [comment, marked as answer]
Something prevents creating that file.
Try to run the installer as admin. Or pause your security software.

**Comment by ardee00** [comment]
Thank you ge0rdi. I usually think to run as Admin but apparently hadn't done so here. When I do, the installer runs fine. Appreciate  your help!
